config/checkstyle.xml

```diff
@@ -117,16 +101,18 @@
 
         <!-- Checks for common coding problems               -->
         <!-- See http://checkstyle.sf.net/config_coding.html -->
-        <module name="AvoidInlineConditionals"/>
+        <!--module name="AvoidInlineConditionals"/-->
         <module name="CovariantEquals"/>
         <module name="EmptyStatement"/>
         <module name="EqualsAvoidNull"/>
         <module name="EqualsHashCode"/>
+        <!--module name="HiddenField"/-->
```

Comment:
Are these rules no longer supported, or are we encountering errors now? If they aren't supported, let's see if we can replace them with some new rules, or if we can't, let's remove them from here.